client/js/components/ratings/rating.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import ActionGrade from 'material-ui/lib/svg-icons/action/grade';
+import RatingActions from '../../actions/rating-actions';
+import Star from './star';
+import UserStore from '../../stores/user-store';
+
+export default class Rating extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleMouseOver = this.handleMouseOver.bind(this);
```

Comment:
@dwoznicka we generally try to follow the convention that function which is passed as a callback and need to be binded we try to name with ```_``` on the begining